analyze/edge-light.ts

```diff
@@ -40,9 +50,22 @@ async function moduleFromPath(path: string): Promise<WebAssembly.Module> {
   throw new Error(`Unknown path: ${path}`);
 }
 
-async function init(context: AnalyzeContext) {
+async function init<Custom extends string>(
+  context: AnalyzeContext,
+  detect?: CustomDetect<Custom>,
+) {
   const { log } = context;
 
+  const convertedDetect = (tokens: string[]) => {
+    if (detect !== undefined) {
+      return detect(tokens).map(
+        (e) => e && ConvertProtocolEntitiesToAnalyzeEntities,
```

Comment:
See above. In my local version, I replaced this completely with a `typeof detect === "function" ? detect : noopDetect`